tests/appsec/test_remoteconfiguration.py

```diff
@@ -121,3 +123,15 @@ def test_rc_activation_validate_products(mock_check_remote_config_enable_in_agen
     appsec_rc_reload_features(tracer)(None, rc_config)
 
     assert RemoteConfig._worker._client._products["ASM_DATA"]
+
+
+def test_rc_rules_data(tracer):
+    tracer.configure(appsec_enabled=True)
```

Comment:
We should avoid reconfiguring the global tracer, these configurations can leak to other tests.

```suggestion
    with override_global_config(dict(_appsec_enabled=True)):
```

Reply:
This change will be made in a follow up PR